src/example-button.module/fields.json

```diff
@@ -118,12 +118,10 @@
                 },
                 "inherited_value": {
                   "property_value_paths": {
-                    "color": "module.styles.group_text.font.color",
-                    "fallback": "module.styles.group_text.font.fallback",
-                    "font": "module.styles.group_text.font.font",
-                    "font_set": "module.styles.group_text.font.font_set",
-                    "styles": "module.styles.group_text.font.styles",
-                    "variant": "module.styles.group_text.font.variant"
+                    "fallback": "theme.heading_font.fallback",
+                    "font": "theme.heading_font.font",
+                    "font_set": "theme.heading_font.font_set",
+                    "variant": "theme.body_font.variant"
```

Comment:
Looks like this is referencing some different values than the [non-hover text group above](https://github.com/HubSpot/marketplace-example-modules/blob/1706566f36272b19ef325f9ba1e71c37fa4a09f4/src/example-button.module/fields.json#L31-L41). My guess is that you would want the hover and non-hover to be the same by default. The way that you have it set up here the button would start as the body font and then swap to the heading font on hover. That might be a bit jarring.

If you want them to be the same by default I think the best way to go about it would be to keep this set as it was originally. Ie

```
  "color": "module.styles.group_text.font.color",
  "fallback": "module.styles.group_text.font.fallback",
  "font": "module.styles.group_text.font.font",
  "font_set": "module.styles.group_text.font.font_set",
  "styles": "module.styles.group_text.font.styles",
  "variant": "module.styles.group_text.font.variant"
```

That way the non-hover fields are referencing the theme settings and the hover fields are referencing the non-hover fields. This would make it so that if the user edited the non-hover font, the hover font should swap to match it, which would likely provide a better user experience.

Reply:
Yeah that makes sense